### ISSUE TITLE
Purchase/Sales  Tax Charges not getting Calculated 

### ISSUE THREAD

**Issue by VARDHANA87** [resilient-tech/india-compliance#386]
The Version Update to 14 we have checked the Sales and Purchase Cycle all the Stock Ledger entry and G.l entry is posted correct, but the Taxes are not calculating.

 

Is Tax Category Defined – Yes  .

Is Item Tax Templet – Yes .

 

Is Purchase Taxes and Charges Calculation – NO .
Is Sales Taxes and Charges Calculation – NO 

<img width="656" alt="image" src="https://user-images.githubusercontent.com/79317458/206958617-e120cef3-b440-41e6-9d88-b6a30d5e505f.png">

Also 

Unable to "Enable": Automatically Add Taxes and Charges from Item Tax Template

<img width="648" alt="image" src="https://user-images.githubusercontent.com/79317458/206958662-130a4dbb-cfa4-4f3a-bbe0-6c0dd64a06c3.png">

**Comment by VARDHANA87** [comment]
Had to get fixed by an Our own developer. The issue was not addressed by the @resilient-tech.

**Comment by vorasmit** [comment]
Hello,

We did not respond for the following reasons:
- Frappe Team was in touch with us for this exact issue as faced by lifelong and we explained to them how they can get along with this over a 30 min call. As they did not revert with any issues we considered this as resolved at your end.
- There are detailed discussions on the forum around this that deals with this.
- There is open PR to solve issues and complexities around this. #223 (sorry for the backlogs on our end)

PS:  App is free and open-source, but consultation/support/customisation is chargeable. 🙏
